### PR TITLE
Adding FreshPorts.md

### PR DESCRIPTION
**Pull request by dlangille**

### CODE REVIEW

File: 2022q4/FreshPorts.md
Comment (pauamma) [inline 2022q4/FreshPorts.md:29]: Missing period.

File: 2022q4/FreshPorts.md
Comment (pauamma) [inline 2022q4/FreshPorts.md:8]: Maybe my taste, but I would remove the commas.

File: 2022q4/FreshPorts.md
Comment (pauamma) [inline 2022q4/FreshPorts.md:28]: Clarification: do you want to merge several SVN repos into a single git repo?

File: 2022q4/FreshPorts.md
Comment (pauamma) [inline 2022q4/FreshPorts.md:14]: s/developers/maintainers/?

File: 2022q4/FreshPorts.md
Comment (pauamma) [inline 2022q4/FreshPorts.md:31]: ":" instead of "." maybe?

File: 2022q4/FreshPorts.md
Comment (pauamma) [inline 2022q4/FreshPorts.md:58]: Missing period.

File: 2022q4/FreshPorts.md
Comment (pauamma) [inline 2022q4/FreshPorts.md:1]: You need to put your own text here. Maybe "FreshPorts: past, present, and future"?

**Review by pauamma — CHANGES_REQUESTED** [review]

File: 2022q4/FreshPorts.md
Comment (dlangille) [reply 2022q4/FreshPorts.md:1]: Yes

File: 2022q4/FreshPorts.md
Comment (dlangille) [reply 2022q4/FreshPorts.md:14]: Yes

File: 2022q4/FreshPorts.md
Comment (dlangille) [reply 2022q4/FreshPorts.md:28]: Multiple. Thanks.

File: 2022q4/FreshPorts.md
Comment (dlangille) [reply 2022q4/FreshPorts.md:29]: I actively try avoid a period after a URL for fear of 404s.

File: 2022q4/FreshPorts.md
Comment (dlangille) [reply 2022q4/FreshPorts.md:31]: yes

File: 2022q4/FreshPorts.md
Comment (dlangille) [reply 2022q4/FreshPorts.md:58]: fixed.

File: 2022q4/FreshPorts.md
Comment (pauamma) [reply 2022q4/FreshPorts.md]: How about `https://github.com/FreshPorts[https://github.com/FreshPorts].`?

Comment (pauamma) [reply 2022q4/FreshPorts.md]: (Since there are other bare URLs and it's a mechanical change that doesn't change content, I can take care of them after or while merging.)

File: 2022q4/FreshPorts.md
Comment (pauamma) [inline 2022q4/FreshPorts.md:34]:
```suggestion
* ingress (code for the back end)
```

For consistency with the earlier unhyphenated spelling.

**Review by pauamma — APPROVED** [review]
A few nits, but I can take care of those when merging if you're OK with them.

**Comment by dlangille** [comment]
> A few nits, but I can take care of those when merging if you're OK with them.

Please proceed, thank you.

**Pull request merged**
